workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -33,75 +36,66 @@
 import org.apache.commons.cli.ParseException;
 
 /**
- * A workload generator for the finance sample app that creates traffic by sending gRPC requests to
- * the server.
+ * An executable method for a workload generator for the finance sample app that creates traffic by
+ * sending gRPC requests to the server.
  */
 public final class WorkloadMain {
-  private static final String DEFAULT_ACCOUNT_BALANCE = "10000";
-  private static final String DEFAULT_TRANSFER_AMOUNT = "20";
   private static final Logger logger = Logger.getLogger(WorkloadMain.class.getName());
 
   private static class WorkloadGenerator {
     private final ManagedChannel channel;
-    private final List<ByteString> ids = new ArrayList<>();
+    private static final int DEFAULT_MAX_QUEUE_SIZE = 100;
```

Comment:
Since the number of concurrent workers is controlled by `threadCount` why keep so many tasks in the queue? Put another way, would we get the same performance with a queue size of 1?

Reply:
Since we have the small loop to check queue size, I would imagine that the queue would tend to stay near capacity. In this case, we could even consider a solution where we don't have a queue (but then we'd have to consider different logic for checking if we should add a new task). We could limit this queue size to something smaller regardless. I can set this to a smaller number and then create an issue for exploring different queue sizes and strategies, or I can look into this more now. Thoughts?

Reply:
> I can set this to a smaller number and then create an issue for exploring different queue sizes and strategies, or I can look into this more now

I don't mean for this to be a blocking comment, this just looks to me like some evidence that we don't really need the task queue.

Reply:
https://github.com/GoogleCloudPlatform/cloud-spanner-samples/issues/54

Reply:
#55